test/integration/scheduler/podsready/scheduler_test.go

```diff
@@ -17,74 +17,92 @@ limitations under the License.
 package podsready
 
 import (
+	"context"
+	"path/filepath"
 	"time"
 
+	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/onsi/ginkgo/v2"
 	"github.com/onsi/gomega"
 	corev1 "k8s.io/api/core/v1"
 	apimeta "k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
 
 	kueue "sigs.k8s.io/kueue/apis/kueue/v1alpha2"
+	"sigs.k8s.io/kueue/pkg/util/pointer"
 	"sigs.k8s.io/kueue/pkg/util/testing"
+	utiltesting "sigs.k8s.io/kueue/pkg/util/testing"
+	"sigs.k8s.io/kueue/test/integration/framework"
 	"sigs.k8s.io/kueue/test/util"
 )
 
+var ignoreCQConditions = cmpopts.IgnoreFields(kueue.ClusterQueueStatus{}, "Conditions")
+
 // +kubebuilder:docs-gen:collapse=Imports
 
-var _ = ginkgo.Describe("SchedulerWithWaitForPodsReady", func() {
+func beforeEachWithTimeout(podsReadyTimeout time.Duration) {
+	fwk = &framework.Framework{
+		ManagerSetup: func(mgr manager.Manager, ctx context.Context) {
+			managerAndSchedulerSetupWithTimeout(mgr, ctx, podsReadyTimeout)
+		},
+		CRDPath:     filepath.Join("..", "..", "..", "..", "config", "components", "crd", "bases"),
+		WebhookPath: filepath.Join("..", "..", "..", "..", "config", "components", "webhook"),
+	}
+	ctx, cfg, k8sClient = fwk.Setup()
 
-	var (
-		ns *corev1.Namespace
-	)
-
-	ginkgo.BeforeEach(func() {
-		ns = &corev1.Namespace{
-			ObjectMeta: metav1.ObjectMeta{
-				GenerateName: "core-",
-			},
-		}
-		gomega.Expect(k8sClient.Create(ctx, ns)).To(gomega.Succeed())
-	})
+	defaultFlavor = utiltesting.MakeResourceFlavor("default").Obj()
+	gomega.Expect(k8sClient.Create(ctx, defaultFlavor)).To(gomega.Succeed())
 
-	ginkgo.When("Scheduling workloads on clusterQueues", func() {
-		var (
-			prodClusterQ *kueue.ClusterQueue
-			devClusterQ  *kueue.ClusterQueue
-			prodQueue    *kueue.LocalQueue
-			devQueue     *kueue.LocalQueue
-		)
+	ns = &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "core-",
```

Comment:
match the suite's name

```suggestion
			GenerateName: "podsready-",
```

Reply:
Done